care.config.ts

```diff
@@ -15,9 +16,7 @@ interface ILogo {
 }
 
 const boolean = (key: string, fallback = false) => {
-  if (env[key] === "true") return true;
-  if (env[key] === "false") return false;
-  return fallback;
+  return booleanFromString(env[key], fallback);
```

Comment:
hmm, we can remove boolean fn and replace its usages with booleanFromString, no? 🤔